src/inverseRobot.js

```diff
@@ -7,7 +7,21 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const values = Object.keys(robot);
+  const keys = Object.values(robot);
+  let newBot = {};
+
+  for (let i = 0; i < values.length; i++) {
```

Comment:
it would be much more concise if you use the for in loop:
`for (const key in robot) {` which iterates through object keys
in such a way swapping is as easy as: `newRobot[robot[key]] = key`